homework2/task01/project1/task1.py

```diff
@@ -0,0 +1,94 @@
+"""
+Given a file containing text. Complete using only default collections.
+
+1) Find 10 longest words consisting from largest amount of unique symbols
+2) Find rarest symbol for document
+3) Count every punctuation char
+4) Count every non ascii char
+5) Find most common non ascii char for document
+"""
+from typing import List
+
+
+def file_presettings(file_path: str) -> str:
+    """
+    Return unpacked file, encode in readable format.
+
+    First sub() deletes '\n' between lines. Second sub() remove punctuation.
```

Comment:
sub ?

Reply:
Yes, sorry, I forgot to change it.
Fixed